pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -354,11 +354,6 @@ func ValidatePriorityLevelConfiguration(pl *flowcontrol.PriorityLevelConfigurati
 	return allErrs
 }
 
-// ValidatePriorityLevelConfigurationUpdate validates the update of priority-level-configuration.
-func ValidatePriorityLevelConfigurationUpdate(old, pl *flowcontrol.PriorityLevelConfiguration) field.ErrorList {
```

Comment:
huh, weird that this isn't called, seems like a bug

Reply:
[`66fc0d7` (#112306)](https://github.com/kubernetes/kubernetes/pull/112306/commits/66fc0d703794f309c9715028d3b63f64c281a5fd#r978608952)
look like it was removed 19 days ago.